src/test/java/com/purbon/kafka/topology/integration/SaslPlaintextKafkaContainer.java

```diff
@@ -0,0 +1,142 @@
+package com.purbon.kafka.topology.integration;
```

Comment:
Can you move this to a dedicated container utility package? so it is not mixed with the IT tests?
Thanks a lot.

Reply:
Moved to ...integration.containerutils